chromium_src/chrome/browser/ui/tabs/public/tab_features.h

```diff
@@ -8,11 +8,12 @@
 
 #include "base/callback_list.h"
 
-#define Init(...)                 \
-  Init_ChromiumImpl(__VA_ARGS__); \
-  virtual void Init(__VA_ARGS__)
+#define TabFeatures TabFeatures_Chromium
 
 #include "src/chrome/browser/ui/tabs/public/tab_features.h"  // IWYU pragma: export
-#undef Init
+
+#undef TabFeatures
```

Comment:
this actually turns out to be a problem

<img width="980" alt="image" src="https://github.com/user-attachments/assets/14ad52b5-c31b-44ea-b7b0-40eb00955a14" />